packages/lib/src/toolbar/controls/ExportEntry.tsx

```diff
@@ -0,0 +1,63 @@
+import styles from './Selector/Selector.module.css';
+
+interface Props {
+  format: string;
+  url: URL | (() => Promise<URL | Blob>) | undefined;
+}
+
+function ExportEntry(props: Props) {
+  const { format, url } = props;
+  const label = `Export to ${format.toUpperCase()}`;
+  const filename = `data.${format}`;
+
+  async function handleBtnClick(getURLOrBlob: () => Promise<URL | Blob>) {
```

Comment:
This function could be moved below the check `url instanceof URL` as it will not be used in this case

Reply:
Functions declarations are hoisted, so this won't do what you think 😉: https://developer.mozilla.org/en-US/docs/Glossary/Hoisting